src/main.py

```diff
@@ -181,18 +182,12 @@ def do_startup(self):
         Gtk.Application.do_startup(self)
         self.__register_actions()
 
-        # Opt-in to follow dark mode user preference.
-        # TODO Remove after porting to libadwaita.
-        manager = Handy.StyleManager.get_default()
-        manager.set_color_scheme(Handy.ColorScheme.PREFER_LIGHT)
-
-        # TODO AdwApplication manages this automatically for us.
         self.dark_provider = Gtk.CssProvider()
```

Comment:
do we need this?

Reply:
No more needed I think

Reply:
hm, it doesn't seem like the styles are automatically loaded

Reply:
ah, I see, we didn't really call AdwApplication